File: CodingChallengeSolutionsC++/1534. Count Good Triplets/1534.CountGoodTriplets.py
class Solution(object):
    def countGoodTriplets(self, arr, a, b, c):
        counter = 0
        for x in range(len(arr)-2):
            for y in range(x+1,len(arr)-1):
                for z in range(y+1,len(arr)):
                    if abs(arr[x]-arr[y]) <= a and abs(arr[y]-arr[z]) <= b and abs(arr[x]-arr[z]) <= c:
                        counter+=1
        return counter

File: CodingChallengeSolutionsC++/1534. Count Good Triplets/1534.CountGoodTriplets.js
var countGoodTriplets = function (arr, a, b, c) {
    counter = 0
    for (x = 0; x < arr.length; ++x) {
        for (y = x + 1; y < arr.length; ++y) {
            for (z = y + 1; z < arr.length; ++z) {
                if (Math.abs(arr[x] - arr[y]) <= a && Math.abs(arr[y] - arr[z]) <= b && Math.abs(arr[x] - arr[z]) <= c) {
                    counter++;
                }
            }
        }
    }
    return counter
};